pleskdistup/actions/distupgrade.py

```diff
@@ -145,6 +146,135 @@ def estimate_revert_time(self) -> int:
+        ret = [sources_list_path]
+        for root, _, filenames in os.walk(sources_list_d_path):
+            for f in filenames:
+                if f.endswith(".list"):
+                    ret.append(os.path.join(root, f))
+        return ret
+
+    def __init__(self,
+        sources_list_path: str = "/etc/apt/sources.list",
+        sources_list_d_path: str = "/etc/apt/sources.list.d/",
+    ) -> None:
+        self._name = "Check repo.list leftovers from previous conversion"
+        self.description = "These repo.list files are found with backups, please check/restore them, and delete the backups: {}"
+        self.sources_list_path = sources_list_path
+        self.sources_list_d_path = sources_list_d_path
+
+    @property
+    def name(self) -> str:
+        return self._name.format(self)
+
+    @name.setter
+    def name(self, val: str) -> None:
+        self._name = val
+
+    def _do_check(self) -> bool:
+        log.debug(f"Checking leftover repo.list files")
+        archived_files = [f for f in
+            CheckAptReposBackups.get_all_repo_list_files(self.sources_list_path, self.sources_list_d_path)
+                if files.backup_exists(f)]
+        if archived_files:
+            self.description = self.description.format(",".join(archived_files))
+            return False
+        return True
+
+
+class ReplaceAptReposRegexp(action.ActiveAction):
+    from_regexp: str
+    to_regexp: str
+    sources_list_path: str
+    sources_list_d_path: str
+    _name: str
+
+    def __init__(
+        self,
+        from_regexp: str,
+        to_regexp: str,
+        sources_list_path: str = "/etc/apt/sources.list",
+        sources_list_d_path: str = "/etc/apt/sources.list.d/",
+        name: str = "set up APT repositories to change from {self.from_regexp!r} to {self.to_regexp!r}",
+    ) -> None:
+        self.from_regexp = from_regexp
+        self.to_regexp = to_regexp
+        self.sources_list_path = sources_list_path
+        self.sources_list_d_path = sources_list_d_path
+
+        self._name = name
+
+    @property
+    def name(self) -> str:
+        return self._name.format(self=self)
+
+    @name.setter
+    def name(self, val: str) -> None:
+        self._name = val
+
+    def _apply_replace_to_file(self, fpath: str, ptrn: re.Pattern, to_regexp: str) -> None:
+        changed = False
+        new_lines = []
+        with open(fpath) as f:
+            for line in f:
+                new_lines.append(ptrn.sub(to_regexp, line))
+                if new_lines[-1] != line:
+                    changed = True
+        if not changed:
+            return
+        files.backup_file(fpath)
+        with open(fpath, 'w') as f:
+            f.writelines(new_lines)
+
+    def _get_all_repo_list_files(self) -> typing.List[str]:
+        return CheckAptReposBackups.get_all_repo_list_files(self.sources_list_path,
+                    self.sources_list_d_path)
```

Comment:
Too much indent. I suggest opening a new line after '(' and use single indent afterwards, that is:
```python
return f(
    arg1,
    arg2,
)
```